.eslintrc.json

```diff
@@ -11,6 +11,7 @@
     "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
     "react/prefer-stateless-function": 0,
     "react/forbid-prop-types": 0,
-    "function-paren-newline": 0
+    "function-paren-newline": 0,
+    "arrow-body-style": 2
```

Comment:
Is `2` a valid value for this property? Seem to be `always`, `as-needed` and `never`: https://eslint.org/docs/rules/arrow-body-style

Reply:
I thought that too but when running it with "never" as a string returns;

<img width="907" alt="screen shot 2017-10-18 at 14 28 15" src="https://user-images.githubusercontent.com/505570/31721230-bbd1ac54-b410-11e7-94c3-5e8952dcd6a0.png">